src/components/search/searchHeader/SearchHeader.tsx

```
@@ -6,17 +6,18 @@ import useNavigateTo from '@hooks/useNavigateTo';
 
-const SearchHeader = ({ searchText, prevPath }: SearchHeader) => {
+const SearchHeader = ({ searchText, prevPath, autoFocus = false }: SearchHeader) => {
   const handleToBack = useNavigateTo(prevPath);
 
   return (
     <header className={styles.searchHeaderStyle}>
       <button className={styles.backBtn} onClick={handleToBack}>
         <Icon.IcnArrowBlackLeft />
       </button>
-      <SearchBar searchText={searchText} />
+      <SearchBar searchText={searchText} autoFocus={autoFocus} />
```

ESLint

[error] The autoFocus prop should not be used, as it can reduce usability and accessibility for users.

(jsx-a11y/no-autofocus)